app/helpers/accounts_helper.rb

```diff
@@ -5,12 +5,7 @@ def summary_card(title:, &block)
   end
 
   def sync_path_for(account)
-    if account.plaid_account_id.present?
-      sync_plaid_item_path(account.plaid_account.plaid_item)
-    elsif account.simplefin_account_id.present?
-      sync_simplefin_item_path(account.simplefin_account.simplefin_item)
-    else
-      sync_account_path(account)
-    end
+    # Always use the account sync path, which handles syncing all providers
```

Comment:
This is such a relief to see ... that whole `elsif` train was starting to worry me! 🎉